type_default.go

```diff
@@ -349,12 +382,28 @@ func (t *intType) NewColumnReader(columnIndex, bufferSize int) ColumnReader {
 	}
 }
 
+func (t *intType) ReadDictionary(columnIndex, numValues int, decoder encoding.Decoder) (Dictionary, error) {
+	if t.IsSigned {
+		if t.BitWidth == 64 {
+			return readInt64Dictionary(t, makeColumnIndex(columnIndex), numValues, decoder)
+		} else {
+			return readInt32Dictionary(t, makeColumnIndex(columnIndex), numValues, decoder)
+		}
+	} else {
```

Comment:
You can drop the else here because both sides of the if statement return.

Reply:
True, but it loses symmetry in the method body, which I find to be a good way to convey intent as somewhat related conditions align on the same indentation levels.